tools/test.py

```diff
@@ -697,6 +697,7 @@ def Execute(args, context, timeout=None, env={}, faketty=False):
     fd_in = fd_err = fd_out
     pty_out = out_master
   else:
+    fd_in = None
```

Comment:
@thefourtheye Hmm, sorry. I thought `fd_in = None` would actually close the fd, but it doesn’t appear to do that… so yes, this line can probably be removed.